src/Nirum/Targets/Python.hs

```diff
@@ -1069,6 +1069,22 @@ class #{className}(#{T.intercalate "," $ compileExtendClasses annotations}):
 
     __nirum_type__ = 'union'
 
+    __nirum_annotations__ = map_type({
+%{ forall (annoIdentifier, annoArgSet) <- M.toList $ A.annotations annotations }
+        '#{toAttributeName annoIdentifier}': map_type({
+%{ forall (annoArgIdent, annoArg) <- M.toList annoArgSet }
+            '#{toAttributeName annoArgIdent}':
+%{ case annoArg }
+%{ of AI.Text t }
+                #{stringLiteral t},
```

Comment:
It looks like `stringLiteral` cannot deal with multiline text. Since document string on Nirum IDL can be multiline text, I think we have to use docstring to compile it appropriately like below (referenced https://github.com/nirum-lang/nirum/blob/cb71cf18e9e76088f597156ea188ce5883480b6e/src/Nirum/Targets/Python/CodeGen.hs#L306-L325 )

```
quotedLiteral :: Text -> Text -> Code
quotedLiteral quote string =
    open $ Data.Text.concatMap esc string `snoc` quote
  where
    openQuote :: Text
    openQuote =
        if Data.Text.any (> '\xff') string
        then "u" ++ quote
        else  quote
    open :: Text -> Text
    open =  Data.Text.append openQuote
    esc :: Char -> Text
    esc '"' = "\\\""
    esc '\\' = "\\\\"
    esc '\t' = "\\t"
    esc '\n' = "\\n"
    esc '\r' = "\\r"
    esc c
        | c >= '\x10000' = pack $ printf "\\U%08x" c
        | c >= '\xff' = pack $ printf "\\u%04x" c
        | c < ' ' || c >= '\x7f' = pack $ printf "\\x%02x" c
| otherwise = Data.Text.singleton c

stringLiteral :: Text -> Code
stringLiteral = quotedLiteral "\""

docstringLiteral :: Text -> Code
dostringLiteral = quotedLiteral "\"\"\""
```

Reply:
I was miss read it. It works fine